charon/tests/ui/constants.out

```diff
@@ -223,8 +223,8 @@ fn test_crate::get_z2() -> i32
     let @0: i32; // return
     let @1: i32; // anonymous local
     let @2: i32; // anonymous local
-    let @3: i32; // anonymous local
-    let @4: i32; // anonymous local
+    let @3: fn() -> i32; // anonymous local
+    let @4: fn() -> i32; // anonymous local
```

Comment:
I have _no idea_ what's happening here. It seems that every local that is assigned a constant value now things it's a function type?

Reply:
Oh lord, this type is the same type as the enclosing function (reproducibly), and comes from `ConstantExpr.ty`